src/i18n/es.json

```diff
@@ -410,5 +410,8 @@
   "userMetadata.userType": "Tipo de usuario",
   "userMetadata.hasFinishedInitialProfileSync": "Incorporación finalizada",
   "userMetadata.isMigratedFromCircles": "Migrado de Círculos v1",
+  "userMetadata.localeLanguage": "Idioma",
+  "userMetadata.businessType": "Business Type",
```

Comment:
Did we want to customize these or it's fine to leave it as the English versions?

Reply:
Nope, you're right. I forgot to ask for translations on these!